src/main/java/uk/gov/justice/hmpps/datacompliance/client/duplicate/detection/AwsAthenaDuplicateDetectionClient.java

```diff
@@ -113,6 +113,12 @@ private void wait(final AtomicInteger numberOfAttempts) {
         return  resultPage.resultSet().rows().stream()
                 .map(DuplicateOffenderRow::new)
                 .filter(not(DuplicateOffenderRow::isHeaderRow))
-                .map(row -> new DuplicateResult(row.getComplementOf(referenceOffender), row.getMatchScore()));
+                .map(row -> new DuplicateResult(
+                        row.getComplementOf(referenceOffender),
+                        matchScoreToPercentageConfidence(row.getMatchScore())));
+    }
+
+    private double matchScoreToPercentageConfidence(final double matchScore) {
+        return matchScore * 100;
     }
```

Comment:
Using a percentage rather than a fraction to be consistent with the image confidence percentage